build/csisamples/hostpath/deploy/csinode.yaml

```diff
@@ -0,0 +1,84 @@
+kind: DaemonSet
+apiVersion: apps/v1beta2
+metadata:
+  name: csi-hostpath-edge
+spec:
+  selector:
+    matchLabels:
+      app: csi-hostpath-edge
+  template:
+    metadata:
+      labels:
+        app: csi-hostpath-edge
+    spec:
+      hostNetwork: true
+      containers:
+        - name: node-driver-registrar
+          image: quay.io/k8scsi/csi-node-driver-registrar:v1.1.0
+          imagePullPolicy: IfNotPresent
+          lifecycle:
+            preStop:
+              exec:
+                command: ["/bin/sh", "-c", "rm -rf /registration/csi-hostpath /registration/csi-hostpath-reg.sock"]
+          args:
+            - --v=5
```

Comment:
why not default log level ?

Reply:
Let's keep it as the other csi drivers do:)